Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+cryptography = "*"
+jwcrypto = "*"
+PyYAML = "*"
+"e1839a8" = {path = ".", editable = true}
```

Comment:
What is this?!

Reply:
Guessing this was to get setuptools to be installed. Don't understand the name though.

Reply:
Take a look inside setup.py.  There are 2 links to guides I was using around this.
It'd be good to have someone else verify that I'm using it right as I've never worked on a library before (only applications in the past)

Reply:
Yeah, I see why it's there. I don't understand the name though. Can you change this to `sdc_cryptography` or similar so that it's clear?

Reply:
Presumably you made it by doing something like `pipenv install -e .` though. I think you should be able to just change the name. I'll test it.

Reply:
Though I'm not really sure how you generated it. Deleting that line and running `pipenv install -e .` adds the following line to my Pipfile:

`git@github.com:ONSdigital/sdc-cryptography.git`

Reply:
What version of pipenv are you using?  I tried that and only got the short hash version

Reply:
I upgraded my pipenv and it worked ^^